test/e2e/common/node/container_probe.go

```diff
@@ -459,7 +459,7 @@ var _ = SIGDescribe("Probing container", func() {
 		Testname: Set terminationGracePeriodSeconds for livenessProbe
 		Description: A pod with a long terminationGracePeriod is created with a shorter livenessProbe-level terminationGracePeriodSeconds. We confirm the shorter termination period is used.
 	*/
-	ginkgo.It("should override timeoutGracePeriodSeconds when LivenessProbe field is set [Feature:ProbeTerminationGracePeriod]", func(ctx context.Context) {
```

Comment:
if you dropping Feature, need to add NodeConformance instead. Otherwise it will be orphaned.

Reply:
> if you dropping Feature, need to add NodeConformance instead. Otherwise it will be orphaned.

Updated.